mail_layout_preview/tests/test_layout_preview.py

```diff
@@ -16,7 +16,7 @@ def _create_template(env, model, **kw):
             "subject": "Preview ${object.name}",
             "body_html": "<p>Hello ${object.name}</p>",
             "model_id": env["ir.model"]._get(model).id,
-            "user_signature": False,
+            # "user_signature": False,
```

Comment:
why this?

Reply:
user_signature has been removed in 14.0 and it is useless for the test, I think

Reply:
just delete it then ;)

Reply:
updated, thanks